packages/angular/src/app/components/profile-card/profile-card.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="panel">
+  <div class="title-text panel-header">
+    {{title}} </div>
+  <div class="form-container">
+    <ng-content select="[top-item]"></ng-content>
+    <dx-form [formData]="cardData"
+             [showColonAfterLabel]="true"
+             [colCount]="(screen.xSmallScreenChanged | async) ? 2 : colCount"
+             [screenByWidth]="getSizeQualifier"
+             (onFieldDataChanged)="dataChanged.emit($event)"
+             labelLocation="top"
+             labelMode="outside"
+    >
```

Comment:
Let's move output to bottom of the attributes:

```suggestion
    <dx-form [formData]="cardData"
             [showColonAfterLabel]="true"
             [colCount]="(screen.xSmallScreenChanged | async) ? 2 : colCount"
             [screenByWidth]="getSizeQualifier"
             labelLocation="top"
             labelMode="outside"
             (onFieldDataChanged)="dataChanged.emit($event)"
    >
```